xcodeproj/BUILD

```diff
@@ -0,0 +1,4 @@
+exports_files([
```

Comment:
This seems needed, otherwise I get:
```
rules_xcodeproj/doc/BUILD.bazel:11:12: no such target '//xcodeproj:experimental.bzl': target 'experimental.bzl' not declared in package 'xcodeproj'; however, a source file of this name exists.  (Perhaps add 'exports_files(["experimental.bzl"])' to xcodeproj/BUILD?) defined by /Users/patrickb/src/rules_xcodeproj/xcodeproj/BUILD and referenced by '//doc:experimental_doc'
```

Reply:
Can we add a comment above it saying it's for Stardoc?

Reply:
Done